api/controllers/project.js

```diff
@@ -1052,8 +1052,14 @@ exports.protectedPut = async function (args, res, next) {
   filteredData.proponent = projectObj.proponent;
   filteredData.currentPhaseName = projectObj.currentPhaseName;
 
-  console.log("Updating with:", filteredData);
-  console.log("--------------------------");
+  // To avoid updating the phaseHistory if the phase hasn't changed
+  if ( filteredData.phaseHistory !== null && JSON.stringify(filteredData.phaseHistory[filteredData.phaseHistory.length-1]) !== JSON.stringify(filteredData.currentPhaseName)){
```

Comment:
I don't think this is going to work. Any of the data that came over in a migration has `phaseHistory` as an array of IDs. I think that only `currentPhaseName._id` will need to be pushed onto the history.  

Thanks for fixing up those console logs.

Reply:
As discussed the currentPhaseName ends up being a String since it only appears as an object once unwound, which addresses your first comment.
 I'll take a look at removing the JSON.stringify if it's unnecessary as discussed..